lib/internal/process/pre_execution.js

```diff
@@ -545,15 +545,20 @@ function initializeDeprecations() {
     });
   }
 
-  if (pendingDeprecation) {
+  if (getOptionValue('--deprecated-process-binding')) {
     process.binding = deprecate(process.binding,
                                 'process.binding() is deprecated. ' +
                                 'Please use public APIs instead.', 'DEP0111');
-
-    process._tickCallback = deprecate(process._tickCallback,
-                                      'process._tickCallback() is deprecated',
-                                      'DEP0134');
+  } else {
+    // Ideally it wouldn't be defined at all already but realm.js does not
+    // provide a means of checking to see if the option is enabled or not,
+    // so we do it here.
+    delete process.binding;
```

Comment:
This should at least throw a warning and tell users about the option `--deprecated-process-binding`.